ares_util/ares.py

```diff
@@ -162,23 +161,19 @@ def build_city(city, address):
     return city or address.split(',')[0].strip()
 
 
-def validate_czech_company_id(business_id):
+def validate_czech_company_id(business_id: str) -> str:
```

Comment:
ARES doesn't return response for `company_id` with 7 places, [example](https://ares.gov.cz/ekonomicke-subjekty-v-be/rest/ekonomicke-subjekty/1603094). This will normalize `1603094` to `01603094`